requirements.txt

```diff
@@ -1,4 +1,5 @@
 createrepo_c~=0.17.0
+django>=2.2.14,<=2.2.20
```

Comment:
maybe `django~=2.2.14,!2.2.21,!2.2.22` can do the trick here.

Reply:
The issue with using the `!...` syntax is that a newer release that can come out and it would bring with it the problematic changes introduced in 2.2.21. I think leaving it as `<=2.2.20` would prevent that better. Nice syntax tho.